hid-xpadneo/src/hid-xpadneo.c

```diff
@@ -1215,6 +1215,9 @@ static const struct hid_device_id xpadneo_devices[] = {
 	{HID_BLUETOOTH_DEVICE(USB_VENDOR_ID_MICROSOFT, 0x02FD)},
 	{HID_BLUETOOTH_DEVICE(USB_VENDOR_ID_MICROSOFT, 0x02E0)},
 
+	/* XBOX ONE Series X / S */
+	{HID_BLUETOOTH_DEVICE(USB_VENDOR_ID_MICROSOFT, 0x0B13)},
```

Comment:
Looks fine but I'd rather move it below `0x0B05`.

Reply:
You may need to add the PID to the "pretend XB1S mode" switch in `xpadneo_input_configured()` if SDL applies wrong mappings after your changes.

Reply:
Or pretend a different firmware version...